projects/storefrontlib/shared/components/popover/popover.component.ts

```diff
@@ -160,18 +160,18 @@ export class PopoverComponent implements OnInit, OnDestroy, AfterViewChecked {
     this.eventSubject.next(PopoverEvent.ESCAPE_KEYDOWN);
   }
 
-  protected isClickedOnPopover(event) {
+  protected isClickedOnPopover(event: any) {
```

Comment:
instead of any, maybe put it as `MouseEvent` since it comes from line 149 on outsideClick which shows that it's a MouseEvent

Reply:
same for the bottom one